### DISCUSSION TITLE
Password Reset Feature

### DISCUSSION THREAD

**Discussion by MrSharpp** [verdaccio/verdaccio#4091] [Ideas]
I just used Verdaccio and personally when i created my new account, i couldn't reset my password.

So this is a crucial feature, for every app.

Should i start implementing it?

**Comment by juanpicado** [comment]
You might start reading the source code https://github.com/verdaccio/verdaccio/blob/master/packages/auth/src/auth.ts#L133 and review what already exist https://github.com/verdaccio/verdaccio/pull/1034 

Also, what exactly are you expecting? just fyi `npm profile set password` already was implemented.
